### PR TITLE
Add player names to the starmap and in-game

### PR DESCRIPTION
**Pull request by Mushroom**
This adds the location of other players to the star map

### CODE REVIEW

File: NebulaPatcher/Patches/Dynamic/UIStarmap_Patch.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿using HarmonyLib;
+using NebulaWorld;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
```
Comment (Baldie-dev) [inline NebulaPatcher/Patches/Dynamic/UIStarmap_Patch.cs:6]: These dependencies are not used here, right?

Comment (Mushroom) [reply NebulaPatcher/Patches/Dynamic/UIStarmap_Patch.cs]: Done

**Review by Baldie-dev — APPROVED** [review]

File: NebulaPatcher/Patches/Dynamic/UIStarmap_Patch.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        {
+            SimulatedWorld.RenderPlayerNameTagsOnStarmap(__instance);
+        }
+
+        [HarmonyPostfix]
+        [HarmonyPatch("_OnClose")]
+        public static void OnClose_Postfix(UIStarmap __instance)
+        {
+            SimulatedWorld.ClearPlayerNameTagsOnStarmap();
+        }
```
Comment (hubastard) [inline NebulaPatcher/Patches/Dynamic/UIStarmap_Patch.cs:21]: Shouldn't this be skipped if not in multiplayer session ?

Comment (Mushroom) [reply NebulaPatcher/Patches/Dynamic/UIStarmap_Patch.cs:21]: Nothing would happen as there would not be any remote players in the list, but I can add a check

**Pull request merged**